IaC/rel.cd/JenkinsStack.yml

```diff
@@ -457,11 +457,11 @@ Resources:
       LaunchTemplateData:
         NetworkInterfaces:
         - DeviceIndex: 0
-          SubnetId: !Ref JSubnetB
+          SubnetId: !Ref JRELSubnetB
           Groups:
-          - !GetAtt JVPC.DefaultSecurityGroup
-          - !GetAtt HTTPSecurityGroup.GroupId
-          - !GetAtt SSHSecurityGroup.GroupId
+          - !GetAtt JRELVPC.DefaultSecurityGroup
+          - !GetAtt HTTPRELSecurityGroup.GroupId
+          - !GetAtt SSHRELSecurityGroup.GroupId
```

Comment:
1. in this template in the first line I see `us-west-1` region, but stack is run in `eu-west-1`
2. line 540 has version `2.222.3` but in fact, on web interface I see `2.235.5`

please fix line 1 and line 540

Reply:
1. Working on it
2. Fixed locally

Reply:
Regarding second point, I suggest to create another PR where it'll be fixed

@delgod

Reply:
Absolutely we can have another PR for these fixed

Reply:
In latest rebase: 0b6f1f3 
Updated first-line comment with proper zone